API/Routes/index.js

```diff
@@ -2,10 +2,11 @@ import express from 'express';
 import IndexRouter from './indexRouter';
 import HomepageRouter from './HomePageRouter';
 import AllmemoriesRouters from './AllMemoriesRouters';
```

Comment:
Please your files should be in Singular
```suggestion
import AllmemoriesRouter from './AllMemoriesRouter';
```

Reply:
Those were the old ones. I will need to correct them later